cmd/simulator/load/funder.go

```diff
@@ -109,7 +111,10 @@ func DistributeFunds(ctx context.Context, client ethclient.Client, keys []*key.K
 	worker := NewSingleAddressTxWorker(ctx, client, maxFundsKey.Address)
 	txFunderAgent := txs.NewIssueNAgent[*types.Transaction](txSequence, worker, numTxs)
 
-	if err := txFunderAgent.Execute(ctx); err != nil {
+	reg := prometheus.NewRegistry()
+	m := metrics.NewMetrics(reg)
+
+	if err := txFunderAgent.Execute(ctx, m); err != nil {
```

Comment:
Can we switch to creating a single instance of metrics to be shared throughout the test instead of creating this once for funding and a separate time for the duration of the test? It seems that these metrics get thrown away and never used

Reply:
sure